use_cases/examples/mimiciv/mortality_decompensation.ipynb

```diff
@@ -0,0 +1,382 @@
+    "1. Tabular: Using tabular data and applying static models.\n",
+    "2. Temporal: Using temporal data as timeseries and applying temporal models.\n",
+    "3. Combined: Using both tabular and temporal data and applying temporal data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cyclops.models.catalog import STATIC_MODELS, TEMPORAL_MODELS\n",
+    "from cyclops.models.predictor import Predictor\n",
+    "from use_cases.data_processors.process import DataProcessor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET = \"mimiciv\"\n",
+    "USE_CASE = \"mortality_decompensation\"\n",
+    "TABULAR_TYPE = \"tabular\"\n",
+    "TEMPORAL_TYPE = \"temporal\"\n",
+    "COMBINED_TYPE = \"combined\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Tabular Processing\n",
+    "\n",
+    "Tabular processing aims to load the queried data, prepares the data for training and testing, and saves the finalized data splits in `4-final` sub-directory of the data directory.\n",
+    "\n",
+    "If the processed data already exists, this step can be skipped."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabular_processor = DataProcessor(DATASET, USE_CASE, TABULAR_TYPE)\n",
+    "tabular_processor.process_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Tabular modeling\n",
+    "\n",
+    "Tabular modeling aims to train a static model on tabular data to predict mortality decompensation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# List of supported static models\n",
+    "STATIC_MODELS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabular_predictor = Predictor(\"lr\", DATASET, USE_CASE, TABULAR_TYPE)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabular_predictor.dataset.n_features"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabular_predictor.dataset.X_train.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabular_predictor.dataset.y_train.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Training the model\n",
+    "xgb_model = tabular_predictor.fit()"
```

Comment:
Maybe can be tabular_model instead of xgb_model since you actually fetch a linear regressor model.